lib/background_worker/logging_concern.rb

```diff
@@ -0,0 +1,17 @@
+require 'active_support/concern'
+
+module BackgroundWorker
+  module LoggingConcern
+    extend ActiveSupport::Concern
+    attr_reader :job_id
+
+    def logger
+      BackgroundWorker.logger
+    end
+
+    def log(message, options = {})
```

Comment:
Looks like QT does not use this method; which is good as AR obviously do not provide it (They do it better with a TaggedLogger implementation)

lets make this method private